webpack/webpack.common.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+
+const DEV_SERVER_PORT = 3000;
```

Comment:
How about using env?

Reply:
I'm not sure that understand what you mean..